backend/pennyme/slack.py

```diff
@@ -0,0 +1,147 @@
+import json
+import os
+from typing import Dict
+
+from flask import FileStorage
+from PIL import Image, ImageOps
+from slack import WebClient
+from slack.errors import SlackApiError
+
+CLIENT = WebClient(token=os.environ["SLACK_TOKEN"])
+IMG_PORT = "http://37.120.179.15:8000/"
+THIS_PATH = os.path.abspath(__file__)
+# Construct paths based on the location of the current script
+PATH_SERVER_LOCATION = os.path.join(
+    os.path.dirname(THIS_PATH), "..", "..", "..", "images", "server_locations.json"
+)
+PATH_MACHINES = os.path.join(
+    os.path.dirname(THIS_PATH), "..", "..", "data", "all_locations.json"
+)
```

Comment:
We could consider to put all paths and constants into a separate config.py or constants.py file. Just as a possible further cleaning step.

Reply:
I totally agree, the purpose of the PR however was just to refactor